components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialActionSheet.h"
+#import "MaterialColorScheme.h"
+
+#import <Foundation/Foundation.h>
+
+@interface MDCActionSheetColorThemer : NSObject
```

Comment:
Missing class docs.

Reply:
I added the docs in #5209

Reply:
Rad - the class itself also needs to have `/** */` docs for generated API docs.

Reply:
Oh yeah sorry I thought you meant the readme, I'll add those right now.